lib/networks.js

```diff
@@ -103,7 +105,17 @@ addNetwork({
     'seed.bitcoinstats.com',
     'seed.bitnodes.io',
     'bitseed.xf2.org'
-  ]
+  ],
+  genesis: new Buffer('010000000000000000000000000000000000000000000000000000000000' +
```

Comment:
I'm using this as a support branch for a PR to `bitcore-node`. I'll add WIP to the title